src/pages/OrderResult/OrderResult.js

```diff
@@ -0,0 +1,113 @@
+import { useState, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import './OrderResult.scss';
+
+const OrderResult = () => {
+  const navigate = useNavigate();
+  const [user, setUser] = useState([]);
+  const [order, setOrder] = useState([]);
+
+  useEffect(() => {
+    fetch('./data/orderUser.json')
+      .then(res => res.json())
+      .then(data => setUser(data.user));
+  }, []);
+
+  useEffect(() => {
+    fetch('./data/orderResult.json')
+      .then(res => res.json())
+      .then(data => setOrder(data.orderResult));
+  }, []);
+
+  return (
+    <div className="orderResult">
+      {user.map(user => (
```

Comment:
```suggestion
      {users.map(user => (
```
가 더 적절해 보입니다. 
다른 PR에서 남겨드린 리뷰내용인데 중복되는것 같아 수정요청만 드리겠습니다!

Reply:
확인하였습니다.